### ISSUE TITLE
show null/empty as occurring value

### ISSUE THREAD

**Issue by JohannaOtt** [halestudio/hale#497]
if a property is not filled, make it visible in the "occurring values" window to let the user know for how many features an attribute is not filled.

**Comment by thorsten-reitz** [comment]
Empty, i.e. an empty string, should appear in the "Occuring values". If it doesn't, that would be unexpected. As to also counting the number of values that are null, I personally don't see that as an occurig value, because there is no value.

**Comment by JohannaOtt** [comment]
Agreed that it is not an occurring value but then it might be helpful to have an extra tab in the properties or anything the like where null values are indicated.
Just stumbled upon this problem again with @annat2022 while working on an alignment.
This is also related to #794 which is slightly different though as it concerns the reader, not data resulting from a mapping.
